### PR TITLE
isPrimitive

### PR DESCRIPTION
**Pull request by FelipeParreira**
[#786](https://github.com/char0n/ramda-adjunct/issues/786)

### CODE REVIEW

**Review by Maushundb — COMMENTED** [review]
Great work! Just a few nits

**Comment by char0n** [comment]
@Maushundb,

@FelipeParreira is no longer responding on his PRs. So with this PRs we have two options here. Online edit of his PR or producing a new PR by us that replaces this one (attribution to Felipe will be added in commit message body).

**Comment by Maushundb** [comment]
@char0n Looks like we have some conflicts to resolve as well so #2 might be the way to go. Happy to make the suggested edits if they're in line with what'd want.

File: types/index.d.ts [hunk 1]
```diff
@@ -62,6 +62,16 @@ declare namespace RamdaAdjunct {
          */
         isBoolean(val: any): val is boolean;
 
+        /**
+         * Checks if input value is a primitive.
+         */
+        isPrimitive(val: any): boolean;
```
[the remark this answers is not in the document]
Comment (char0n) [reply types/index.d.ts:68]: Sure

File: src/isPrimitive.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * @see {@link RA.isNotObj|isNotObj}
+ * @example
+ *
+ * RA.isPrimitive("string"); //=> true
+ * RA.isPrimitive(1); //=> true
+ * RA.isPrimitive(new String("string")); //=> false
+ * RA.isPrimitive(new Number(1)); //=> false
+ */
+
+const isPrimitive = both(
```
[the remark this answers is not in the document]
Comment (char0n) [reply src/isPrimitive.js:29]: We we can add the link via jsdoc `@see` tag, along with the link I've provided in original issue description #786 (in **Additional Context**.

**Comment by char0n** [comment]
> @char0n Looks like we have some conflicts to resolve as well so #2 might be the way to go. Happy to make the suggested edits if they're in line with what you'd want.

Sounds good, if you're up to it, create a new PR which will supersede this one, and we'll do the review there.

**Comment by Maushundb** [comment]
https://github.com/char0n/ramda-adjunct/pull/1722

**Comment by char0n** [comment]
has been superseeded by #1722

**Pull request closed without merging**